web/app/view/CommandDialog.js

```diff
@@ -90,6 +90,33 @@ Ext.define('Traccar.view.CommandDialog', {
                 hidden: true
             }]
         }, {
+            xtype: 'fieldcontainer',
+            reference: 'paramSetTimezone',
+            name: 'attributes',
+            hidden: true,
+
+            items: [{
+                xtype: 'numberfield',
+                fieldLabel: Strings.commandTimezoneOffset,
+                name: 'timezone',
+                minValue: -12,
```

Comment:
I believe the value is in seconds. I guess you would have to change server side if it doesn't accept seconds.